tests/fixtures/blast/blastSettingsConfig.json

```diff
@@ -308,63 +320,75 @@
       "type": "select",
       "options": [
         {
-          "label": "1,-4",
-          "value": "1,-4"
+          "label": "1,-1",
+          "value": "1,-1"
         },
         {
-          "label": "2,-7",
-          "value": "2,-7"
+          "label": "1,-2",
+          "value": "1,-2"
         },
         {
           "label": "1,-3",
           "value": "1,-3"
         },
         {
-          "label": "2,-5",
-          "value": "2,-5"
-        },
-        {
-          "label": "1,-2",
-          "value": "1,-2"
+          "label": "1,-4",
+          "value": "1,-4"
         },
         {
           "label": "2,-3",
           "value": "2,-3"
         },
         {
-          "label": "1,-1",
-          "value": "1,-1"
+          "label": "2,-5",
+          "value": "2,-5"
         },
         {
-          "label": "5,-4",
-          "value": "5,-4"
+          "label": "2,-7",
+          "value": "2,-7"
         },
         {
           "label": "4,-5",
           "value": "4,-5"
-        },
+        },        
         {
-          "label": "N/A",
-          "value": ""
-        }
+          "label": "5,-4",
+          "value": "5,-4"
+        }        
       ]
     },
     "wordsize": {
       "label": "Word size",
       "description": "Word size for wordfinder algorithm",
       "type": "select",
       "options": [
+        {
+          "label": "2",
+          "value": "2"
+        },
+        {
+          "label": "3",
+          "value": "3"
+        },
+        {
+          "label": "4",
+          "value": "4"
+        },
+        {
+          "label": "5",
+          "value": "5"
+        },
         {
           "label": "6",
           "value": "6"
         },
         {
-          "label": "11",
-          "value": "11"
+          "label": "7",
+          "value": "7"
         },
         {
-          "label": "28",
-          "value": "28"
+          "label": "8",
+          "value": "8"
```

Comment:
I guess you added these because most of the programs wordsize range is 2-8?

Reply:
@jyothishnt yes the default is 2-8 and then its overwritten further down for other program.